iOS/update getting-started guide, removed refs to amplify-app (#2888)
aws-amplify/docs: docs/lib/datastore/fragments/ios/getting-started/40_codegen.md

> On iOS the `amplify` CLI offers an Xcode integration that automatically adds the Amplify-specific files to your project.

Reviewer: ```suggestion
The `amplify` CLI is integrated with Xcode and will automatically add Amplify-specific files to your project.
```

Author: I wouldn't change the wording here as unfortunately the CLI is not exactly "integrated" with Xcode (at least not as integrated as it currently is with Android Studio) :)

1. **Run the command**:
    ```
    amplify codegen models
    ```

> 2. Once the command finishes, a new group named `AmplifyModels` should be added to your project and contain the following files:

Reviewer: ```suggestion
2. The `amplify codegen models` command adds a new `AmplifyModels` group to your Xcode project. It will contain the following files:
```
